documentation/popups.md

```diff
@@ -95,6 +97,13 @@ data:
 
 ![Screenshot of a popup rendering the form described above](https://user-images.githubusercontent.com/1299821/182712670-f3b4fdb7-84a9-49d1-a26f-2cdaa450fa0e.png)
 
+**NOTE:** Some Home Assistant selectors may use another dialog for input. [Date](https://www.home-assistant.io/docs/blueprint/selectors/#date-selector) and [Date & Time](https://www.home-assistant.io/docs/blueprint/selectors/#date--time-selector) use a selector dialog for date & time. Browser Mod popups are not in the Home Assistant DOM hierachy so stacking will need to be adjusted by [styling](#styling-popups) the popup with th styles shown below. The Home Assistant top bar has a z-index of 4, so using a z-index of 5 will work in most cases, but your setup may vary so adjust to suit.
```

Comment:
This shouldn't be the users resposibility. Is there any reason this can't be in the default styles?

Reply:
When fixing this, I was not sure of bext z-index to use that would be 100% foolproof, as z-index is not set but determined by DOM order. Since fixing, I note that the sidebar is higher than 5, so may need to use 10?? If we can agree on a z-index then yes, can set the style as default.

Reply:
`ha-dialog` uses `z-index: var(--dialog-z-index, 8);`
Maybe that would be a good thing to copy? Does it affect more-info nesting?

Reply:
Needs to be `z-index: 7`, otherwise the form dialogs are shown behind the popup. more-info nesting works fine with `z-index: 7`. NOTE: For z-index to stick, psoition needs to be set away from static e.g. `position: fixed`. As popups are at top I can't see that to be any issue, though was another reason I did not make any such styling default.

Reply:
Could that reason have to do with dropdown menus in the popup? There's something about that in the back of my head...

Reply:
If you are talking about `position: fixed`, that is just CSS requirement to be able to apply `z-index`

Reply:
@thomasloven I have been looking into this and the more I think about it, the more risky it is to set style as default as it may break styling that others have manually applied. Given that there is a solution here for users I recommend we finalise 2.3.8. I have raised #845